gallery/visualtests/nxToast.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+const { setupBrowser } = require('./testUtils');
+
+describe('NxToast', function() {
+  const openButton = '.nx-btn-bar .nx-btn--secondary';
+  const toastContainer = '.nx-toast-container';
+
+  describe('with global sidebar and header', function() {
```

Comment:
Aside from the `describe` text and the URL, are each one of these blocks totally identical? If they are, I'd create a helper function which removes the code duplication and receives the describe text and URL as parameters.

Reply:
> Aside from the `describe` text and the URL, are each one of these blocks totally identical? If they are, I'd create a helper function which removes the code duplication and receives the describe text and URL as parameters.

Cleaned up the repetition here with a `testNxToast` function : [9ff7776](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/9ff77760ac64c54ae67cbb9631ba2f4a1ae04ae3)